### DISCUSSION TITLE
posts/2024/04/field-essay-48/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#52] [General]
# posts/2024/04/field-essay-48/

村尾休闲：十点半 🔗和昨天一样，下午四点以后，我经过村子的一处转角小道时，见到四个奶奶正围着一张搭着小蓬的桌子一起打牌——昨天提及的慈眉善目的奶奶也在里头。我见过她们中的每一位，遗憾的是一位也叫不上名字。

https://guozheng.rbind.io/posts/2024/04/field-essay-48/

**Comment by earfanfan** [comment]
>我忘记究竟是km婶还是慈眉善目的奶奶，她们中有一人问我想不想吃，我有些受宠若惊，连忙用雷州话说肚子不饿，但让我心中马上有些一紧的是靠在一旁电线杆上的rm婶朝着询问我的大婶/奶奶用雷州话说了一句“你和他说话干嘛？”——我不明所以，当即心中一紧。虽然这不一定意味着有恶意或者任何其他想法，但听起来还是怪怪的。

看到 rm ，我会联想到一个 Linux 命令，remove，删除。

如果允许我发散想象力的话，我觉得让你感到奇怪的这句“你和他说话干嘛？”可能是酱紫的。先交代一下背景，在古早的农村，人们日常休闲娱乐或者聊天是以性别来分群的，大多是男人和男人说话，女人和女人说话，一旦某个妇女和外来的男人说话，有可能会被其他妇女指责为“不守妇道”，在这种情况下，尽管只是向陌生男人表达善意，那位妇女也极有可能会处于时代思想束缚之下的“道德低点”，而其他妇女就会自动处于道德高点而任意指责那位妇女。

虽然从我的视角看，比我小的都是小孩。但是那几位妇女在与你打交道的时候，可能会更多留意到你是个男子，于是乎……

话又说回来，尽管大家在此一刻都活在同一个时空，但是不同经历的人会有不同的思想观念，那几位妇女确实对你是没有恶意的。

**Reply by residualsun1** [reply]
> 看到 rm ，我会联想到一个 Linux 命令，remove，删除。

哇，没想到你还会Linux。

> 如果允许我发散想象力的话，我觉得让你感到奇怪的这句“你和他说话干嘛？”可能是酱紫的。先交代一下背景，在古早的农村，人们日常休闲娱乐或者聊天是以性别来分群的，大多是男人和男人说话，女人和女人说话，一旦某个妇女和外来的男人说话，有可能会被其他妇女指责为“不守妇道”，在这种情况下，尽管只是向陌生男人表达善意，那位妇女也极有可能会处于时代思想束缚之下的“道德低点”，而其他妇女就会自动处于道德高点而任意指责那位妇女。

我从没想到是这个原因，但觉得你讲得很有道理。我当时自认和她相处多了一些，因为准备水仙大王诞辰仪式的时候，她是最主要的负责人，而我时常过去帮忙，她也为此对我一度表达过谢意，我便以为她会像村里的其他老人一样接纳我，待我友善，没想到她的反应中还是有着不少的防备。

> 话又说回来，尽管大家在此一刻都活在同一个时空，但是不同经历的人会有不同的思想观念，那几位妇女确实对你是没有恶意的。

嗯，我也相信她没啥恶意，只是防备，或者还是有意保持距离吧。
